src/generateKeyPair.js

```diff
@@ -0,0 +1,10 @@
+import nacl from 'tweetnacl';
```

Comment:
I guess we no longer need this file, since we dropped the signature field from the on-chain json, right?

Reply:
Removed on d41e940